v2/tests/integration_test.go

```diff
@@ -301,6 +340,12 @@ func TestMain(t *testing.T) {
 	)
 
 	for _, tc := range testcases {
+		// Skip private repo test if required env var not present
+		if tc.name == "GitHub - private repo" &&
+			os.Getenv("BRIGADE_CI_PRIVATE_REPO_SSH_KEY") == "" {
+			continue
+		}
```

Comment:
Is it worth having a more general mechanism that scales better as we (potentially) continue to add test cases that can _only_ run in CI?

I had been thinking we could add a `bool` field to the `testCase` struct like `skipLocally` or something that is `false` by default, but can be set to `true` for the few cases that can only run in CI.

wdyt?

Reply:
But consider this non-blocking.

Reply:
I did consider that, but opted to just wait on adding the toggle until we had more than one test case to utilize it.  Should be quick to add when we get there!

Reply:
(Actually found it handy to be able to run locally _if_ I had the req'd env var set, for testing 😄 )